client/src/AjaxHelper.js

```diff
@@ -8,20 +8,20 @@ class AjaxHelperInstance {
         this.handle4xx = this.handle4xx.bind(this);
     }
 
-    get apiUrl() {
-        return this.url + "/api";
+    get ingestApiUrl() {
```

Comment:
We should also talk to @crhallberg about how we'll manage URLs when we have separate /edit and /ingest routes. Maybe we still need a `get apiUrl()` method that returns simply `this.url`.